src/hlvs_player.cpp

```diff
@@ -77,6 +77,7 @@ class WebotsController : public rclcpp::Node
 
     // Timer and its callback
     // simulation does a step, it does not really make sense to run this in any other frequency
+    // TODO: make rate configurable
```

Comment:
maybe you could add comment here why the 8ms are okay to be hard coded (because it is guaranteed by the rules that simulation does not run faster than real time and with 8ms)